starknet_py/contract.py

```diff
@@ -306,16 +313,18 @@ async def call(
         self,
         *args,
         block_hash: Optional[str] = None,
+        version: Optional[int] = None,
```

Comment:
Does the call need the version as well?

Reply:
Yes, it is needed. It was hard coded to 0, but now we have to support both versions